### DISCUSSION TITLE
Make a declaration

### DISCUSSION THREAD

**Discussion by michaelcattell** [dwp/design-system#439] [Q&A]
Hello, we'd like to use this open discussion on the use of the **Make a declaration**:

**Some discussion points to get things going:**
Have you used or know where this component has been used?
Do you know if there's been any research or testing done on it or around it?
Do you agree with the user need?
Things we don’t know enough?

**Comment by JLOliver-DWP** [comment]
Hello, I am no longer on the Bereavement team, but this declaration regularly used to get very negative feedback, the word 'prosecution' in particular was deemed too harsh, especially for someone recently bereaved. It would be worth contacting Dan O'Connor (Product Owner) or Dinesh Jayachandran (IXD) to get the latest on this. Perhaps by looking at Google Analytic data on services - you could see if there is a high drop-out rate on this page also?

**Comment by a184studio** [comment]
Hello there, Some insights from us. 

**Service: Pension Credit.**

**To Be prosecuted** is the last thing on our list of outcomes, it's also the very last step in the process of DWP trying to sort an overpayment out. 

It heavy handed to assume that people are actively trying to commit fraud and so we should not lead with prosecuted.
We have enough issues at the minute with people not applying at all.
We also see people dropping off at declaration, even with the content attached. 

It challenging enough for applications to manage their change of circumstances, at present we don't make it easy for the citizens to help to understand the process of _managing a claim_ 5+ years after application. No reminders etc etc. 

I think we just need to be very clear about our process, the steps and what possible outcomes are in order of severity.   

**On pension Credit, fraud vs error** 

**Fraud: 27.9%**
Of that: 
Fraud at application: 5.6%
Fraud when in payment is: 94.4%

**Error: 72.1%**
Of that: 
Error at application is: 4.3%
Error when in payment is: 95.7%
 

<img width="1792" alt="Screenshot 2021-09-16 at 11 58 56" src="https://user-images.githubusercontent.com/44495654/133600621-3db3e9d3-36bd-416b-bd34-0ee37cd9a0db.png">
